ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -32,6 +32,8 @@ spec:
       description: Current Certification status of the cert Project
     - name: github_usernames
       description: List of GitHub accounts with permissions allowing operator submission
+    - name: docker_config_json
+      description: Encrypted tokens for partner's registry
```

Comment:
We probably shouldn't use a result to pass this back because it has the potential to be large. Tekton has a hard limit of [4096 bytes](https://github.com/tektoncd/pipeline/blob/main/docs/tasks.md#emitting-results) for any result. Storing this in a workspace would be preferable.

Reply:
ack, I will add it there.

Reply:
Fixed